Frontend/src/components/NoteModal.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import { useState } from "react";
+import { Navigate } from "react-router-dom";
+
+
+// eslint-disable-next-line react/prop-types
+const NoteModal = ({closeModal}) => {
+
+    const [title, setTitle] = useState('')
+    const [description, setDescription] = useState('')
+
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        try{
+            const response = await axios.post("http://localhost:5000/add",
```

Comment:
@NuffSaid-Bore please display a user-friendly error message on the modal if the request fails.
for example; catch (error) {
        setError('Failed to add the task. Please try again .');
        console.log(error);